cfgov/cfgov/settings/base.py

```diff
@@ -264,6 +265,11 @@
     PROJECT_ROOT.child('templates', 'wagtailadmin')
 ]
 
+if 'DJANGO_STATICFILES_IN' in os.environ:
+    pattern = os.path.join(
+        os.environ['DJANGO_STATICFILES_IN'],
```

Comment:
What's the argument over making this configurable vs. just hardcoding use of our current `static.in` as the one place we always place these files? We already document this location, for example [here](https://cfpb.github.io/cfgov-refresh/installation/#webfonts).

Reply:
right now, I don't think the cfgov-refresh version of static.in gets included in the build artifact?

I guess we *could* make it relative, though--   it will always be a sibling to the cfgov directory.

Reply:
Is there any reason why not to modify the Jenkins build to use static.in? In my branch modifying the Jenkins job to use the Docker-based build (docker-dfd-build), I use that and it works well. We don't use it in the non-Docker-based build currently.

Reply:
I don't see a reason not to do that-- I think they just came about at different times, and for different reasons, and nobody ever connected the dots.